testfiles/5ZoneWaterSystems_NoSitePrec.idf

```diff
@@ -0,0 +1,6259 @@
+!-Generator IDFEditor 1.34
+!-Option OriginalOrderTop UseSpecialFormat
+!-NOTE: All comments with '!-' are ignored by the IDFEditor and are generated automatically.
+!-      Use '!' comments if they need to be retained when using the IDFEditor.
+! 5ZoneWaterSystems.idf
```

Comment:
This idf still has a Site:Precipitation object. And this comment line needs the new file name.

Reply:
Thanks. I just fixed the site:precipitation art and the comments in this and the other *_NoSitePrecipitation.idf file